Client/Frontend/Components/FakespotErrorCardView.swift

```diff
@@ -0,0 +1,142 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import UIKit
+import Common
+
+final class FakespotErrorCardView: UIView, ThemeApplicable {
+    private enum UX {
+        static let buttonFontSize: CGFloat = 16
+        static let buttonVerticalInset: CGFloat = 12
+        static let buttonHorizontalInset: CGFloat = 16
+        static let buttonCornerRadius: CGFloat = 13
+        static let containerSpacing: CGFloat = 8
+        static let containerMargins = UIEdgeInsets(top: 8, left: 12, bottom: 8, right: 12)
+        static let iconStackViewSpacing: CGFloat = 4
+        static let horizontalStackViewSpacing: CGFloat = 12
+        static let verticalStackViewSpacing: CGFloat = 4
+        static let iconSize: CGFloat = 24
+        static let titleFontSize: CGFloat = 13
+        static let descriptionFontSize: CGFloat = 13
```

Comment:
This is also 14.

Reply:
<img width="797" alt="Screenshot 2023-08-04 at 17 47 16" src="https://github.com/mozilla-mobile/firefox-ios/assets/26011662/37d68f23-bfc3-4f63-943c-7ca99339c0fd">
Same here.